.github/workflows/build.yml

```diff
@@ -321,19 +321,10 @@ jobs:
           linters: gcc
           run: env LEAN_ABORT_ON_PANIC=1 lake exe runLinter Mathlib
 
-      - name: check environments using lean4checker
-        id: lean4checker
-        run: |
-          git clone https://github.com/leanprover/lean4checker
-          cd lean4checker
-          git checkout v4.8.0-rc1
-          # Now that the git hash is embedded in each olean,
-          # we need to compile lean4checker on the same toolchain
-          cp ../lean-toolchain .
-          lake build
-          ./test.sh
-          cd ..
-          lake env lean4checker/.lake/build/bin/lean4checker
+      # We no longer run `lean4checker` in regular CI, as it is quite expensive for little benefit.
+      # Instead we run it in a cron job on master: see `lean4checker.yml`.
+      # Output is posted to the zulip topic
+      # https://leanprover.zulipchat.com/#narrow/stream/345428-mathlib-reviewers/topic/lean4checker
```

Comment:
I will note that most people don't have access to this stream. Is this bad?

Reply:
I don't have a strong opinion on this, but the bot could post instead on a thread in the nightly testing topic?

Reply:
It's not about nightly testing, so that looks a bit wrong. Maybe we can just post in #mathlib4 ?

Reply:
Sure, I just thought that an "opt-in" thread was better than a thread to which everyone is subscribed.

Reply:
You can always opt out of topics. Maybe topics can even be opt-in?

Reply:
My thinking was to not put it in any high traffic stream, to avoid spamming people. My preference is to leave as is, but of course if anyone expresses a wish to have access to the output we can move it.